docker/docker-compose.yml

```diff
@@ -7,22 +7,9 @@ services:
     ports:
     - "5005:5005"
     volumes:
-    - "./rasa-app-data/models:/app/models"
-    - "./rasa-app-data/config:/app/config"
-    - "./rasa-app-data/project:/app/project"
-    - "./rasa-app-data/projects:/app/projects"
-    - "./rasa-app-data/logs:/app/logs"
-    - "./rasa-app-data/data:/app/data"
+    - "./rasa-app-data/:/app/"
     command:
-    - start
-    - -d
-    - ./model
-    - -c
-    - rest
-    - -u
-    - current/nlu
-    - --endpoints
-    - config/endpoints.yml
+    - run
```

Comment:
This is more of a thought rather than a suggestion, I see this issue working both ways. It appears to me that options would be helpful, to enrich the docker-compose up command by default via ENV variables, like --debug, and especially the --enable-api option, if it would be a single docker file, I wouldn't consider it, but in a compose file is a bit more cumbersome inject all those to the respective service.

Reply:
mhm, I guess everyone has to set up their own docker-compose file anyway, don't you think?